apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -1,12 +1,25 @@
 import { html } from 'lit';
 import type { Meta, StoryObj } from '@storybook/web-components';
-import { PieButton } from '@justeattakeaway/pie-button';
+import { PieButton, BUTTON_TYPE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
```

Comment:
Should it be `BUTTON_TYPES` and `BUTTON_VARIANTS`?

Reply:
It's actually quite common to go with singular naming for Enums. Here's an example of some [guidance from Microsoft](https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN)